jb-yoda/src/main/java/life/jukebox/JBWatcher.java

```diff
@@ -0,0 +1,22 @@
+import java.util.concurrent.*;
+
+public class JBWatcher implements Runnable{
+  private ScheduledFuture<?> watcherHandle;
+  private String jbId;
+
+  public JBWatcher(String jbId) {
+    this.jbId = jbId;
+  }
+
+  public ScheduledFuture<?> getWatcherHandle(){
+    return watcherHandle;
+  } 
+
+  public void setWatcherHandle(ScheduledFuture<?> watcherHandle){
+    this.watcherHandle = watcherHandle;
+  }
+
+  public void run(){
```

Comment:
Javadoc on all non-trivial (basic getters and setters) public methods. Please do this for all.

Reply:
fixed